engine/src/main/java/org/destinationsol/ui/SolInputManager.java

```diff
@@ -147,7 +147,7 @@ public void addScreen(SolApplication solApplication, SolUiScreen screen) {
         screen.onAdd(solApplication);
     }
 
-    private void removeScreen(SolUiScreen screen, SolApplication solApplication) {
+    public void removeScreen(SolUiScreen screen, SolApplication solApplication) {
```

Comment:
Is this access modification potentially problematic at all? Could other things just start randomly removing screens, by chance or on purpose, or is it badly needed for something?

Reply:
Not actually sure what it does in this PR. Probably was just testing some stuff and forgot to revert.